containers/health-check-supportconfig-exporter-image/Dockerfile

```
@@ -0,0 +1,26 @@
+#!BuildTag: uyuni/health-check-supportconfig-exporter:latest
+
+ARG BASE=registry.suse.com/bci/bci-base:15.6
+
+FROM $BASE
+
+RUN zypper -n ref
+RUN zypper -n install python311-PyYAML
```

every RUN is a layer in the image. let's reduce them where we can...

```suggestion
RUN zypper --non-interactive ref && zypper --non-interactive up && \
         zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution \
                       python311-PyYAML
```